unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -968,11 +991,11 @@ aliasMany =
 up :: InputPattern
 up =
   InputPattern
-    "up"
+    "deprecated.up"
     []
     I.Visible
```

Comment:
In the PR description you say these are hidden from help, but I think you forgot to hide them all :)

Reply:
Good catch, thanks :)